tools/methylkit/test-data/input_control1.myCpG.txt

```diff
@@ -0,0 +1,2378 @@
+chrBase	chr	base	strand	coverage	freqC	freqT
+chr21.9826907	chr21	9826907	F	96	18.75	81.25
+chr21.9853326	chr21	9853326	F	16	87.50	12.50
+chr21.9853296	chr21	9853296	F	18	88.89	11.11
+chr21.9860126	chr21	9860126	F	83	100.00	0.00
+chr21.9906663	chr21	9906663	R	14	92.86	7.14
```

Comment:
Yes I meant compare='contains', but you are right for an input this does not matter.